rest/RESTClient.ts

```diff
@@ -80,11 +92,14 @@ export default class RESTClient {
         } else {
           routeBucket.remaining += 1;
         }
-      });
+      };
+
+      if (this.globallyRateLimited) this.queue.push(() => routeBucket.add(call));
+      else routeBucket.add(call);
     });
   }
 
-  calculateRLRoute(endpoint: string, method: HTTP_METHODS) {
+  calculateRLRoute(endpoint: string, method: HTTP_METHODS): string {
```

Comment:
Why is the type return here? Isn't it implied string return anyways?

Reply:
Apparently not. I had to add that in because I was getting errors from something.

Reply:
Also fine on my end